schema_editor/app/scripts/views/recordtype/related-aggregates-directive.js

```diff
@@ -0,0 +1,160 @@
+(function() {
+    'use strict';
+
+    /* ngInject */
+    function RTRelatedAggregatesController(
+        $state, $scope, $stateParams, Schemas, RecordTypes, RecordSchemas, RecordCosts,
+        Notifications
+    ) {
+        var ctl = this;
+        initialize();
```

Comment:
I think it would be good to get into the habit of using Angular's lifecycle hooks now that we know they exist, i.e. this would become `ctl.$onInit = initialize;`